modules/imgproc/src/stb_truetype.cpp

```diff
@@ -0,0 +1,5009 @@
+#include "stb_truetype.hpp"
```

Comment:
License and copyright notice should come first.

Reply:
done